<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Азбука Морзе - Avocado</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(190, 227, 255);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="change_mode">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>0</y>
      <width>321</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Monotxt</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <item>
     <property name="text">
      <string>﹝Азбука Морзе ⟶ Латиница﹞</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>﹝Латиница ⟶ Азбука Морзе﹞</string>
     </property>
    </item>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>50</y>
      <width>552</width>
      <height>470</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label_text">
       <property name="font">
        <font>
         <family>Monotxt</family>
         <pointsize>11</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="text">
        <string>Простой текст</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="text_in_out">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(179, 197, 255);</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_morse">
       <property name="font">
        <font>
         <family>Monotxt</family>
         <pointsize>11</pointsize>
         <weight>50</weight>
         <italic>true</italic>
         <bold>false</bold>
         <strikeout>false</strikeout>
        </font>
       </property>
       <property name="text">
        <string>Код Морзе</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="morse_out">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(179, 197, 255);</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="modeB">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="font">
          <font>
           <family>Monotxt_IV50</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Прием</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>348</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="cleanB">
         <property name="font">
          <font>
           <family>Monotxt_IV50</family>
           <pointsize>11</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Очистить</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Настройка</string>
    </property>
    <addaction name="clean_first"/>
    <addaction name="clean_second"/>
    <addaction name="clean_all"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="our_comand"/>
    <addaction name="app_descrip"/>
    <addaction name="morse_alph"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="our_comand">
   <property name="text">
    <string>Команда</string>
   </property>
  </action>
  <action name="app_descrip">
   <property name="text">
    <string>Приложение</string>
   </property>
  </action>
  <action name="morse_alph">
   <property name="text">
    <string>Азбука Морзе</string>
   </property>
  </action>
  <action name="clean_first">
   <property name="text">
    <string>Очистить первое поле</string>
   </property>
  </action>
  <action name="clean_second">
   <property name="text">
    <string>Очистить второе поле</string>
   </property>
  </action>
  <action name="clean_all">
   <property name="text">
    <string>Очистить все</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
